### PR TITLE
CDS-78132: Moved troubleshooting content from the GitOps installation…

### PR DESCRIPTION
**Pull request by deepakbolangady**
… topic and an internal Confluence topic into a new topic.

Preview: https://6502a75b75adcf1e64732d74--harness-developer.netlify.app/docs/continuous-delivery/gitops/gitops-ref/troubleshooting

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* [CDS-78132](https://harness.atlassian.net/browse/CDS-78132)

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot. 

[CDS-78132]: https://harness.atlassian.net/browse/CDS-78132?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by meenaravichandran1 — APPROVED** [review]

**Review by manavjot-harness — APPROVED** [review]

File: docs/continuous-delivery/gitops/gitops-ref/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### Error: "Agent took too long to respond"
+
+You might encounter the error `the Agent took too long to respond` when installing the Harness GitOps Agent with an existing Argo CD instance.
+
+The error indicates that the Harness GitOps Agent is unable to connect to Redis or to the Argo CD Repo Server and needs additional `NetworkPolicy` settings.
+
+Add the following `podSelector` settings to the `NetworkPolicy` objects defined in your existing Argo CD *argocd-redis* and *argocd-repo-server* services.
+
+The following table lists the `NetworkPolicy` objects for HA and non-HA Agents, and include the YAML before and after the new `podSelector` is added.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/gitops/gitops-ref/troubleshooting.md:20]:
```suggestion
The following table lists the `NetworkPolicy` objects for HA and non-HA Agents, and includes the YAML before and after the new `podSelector` is added.
```

**Review by brian-f-harness — DISMISSED** [review]

**Comment by MarkRamm-harness** [comment]
Looks good.

**Review by MarkRamm-harness — APPROVED** [review]
Looks good

**Comment by deepakbolangady** [comment]
Reviews completed. Force-merging the PR.

**Pull request merged**